Section 3. Hierarchical modelling.ipynb

```diff
@@ -690,7 +690,7 @@
    "metadata": {},
    "source": [
     "### References\n",
-    "1. [The Best Of Both Worlds: Hierarchical Linear Regression in PyMC3](http://twiecki.github.io/blog/2014/03/17/bayesian-glms-3/) by Thomas Wiecki"
+    "1. [The Best Of Both Worlds: Hierarchical Linear Regression in PyMC3](http://twiecki.github.io/blog/2014/03/17/Bayesian-glms-3/) by Thomas Wiecki"
```

Comment:
404 error: http://twiecki.github.io/blog/2014/03/17/Bayesian-glms-3/

Needs to be lowercase it seems: http://twiecki.github.io/blog/2014/03/17/bayesian-glms-3/